.gitignore

```diff
@@ -12,10 +12,17 @@ yarn-debug.log*
 yarn-error.log*
 pnpm-debug.log*
 
-
 # environment variables
 .env
 .env.production
 
 # macOS-specific files
 .DS_Store
+
+# lock files
+package-lock.json
+yarn.lock
```

Comment:
```suggestion
```

I'd like to keep these checked in please. Ensuring the same version of dependencies across builds is desired behaviour.

Reply:
Some more info: https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json